lib/source-code/source-code.js

```diff
@@ -423,7 +423,7 @@ class SourceCode extends TokenStore {
     isSpaceBetweenTokens(first, second) {
         const text = this.text.slice(first.range[1], second.range[0]);
 
-        return /\s/u.test(text.replace(/\/\*.*?\*\//gu, ""));
+        return /\s/u.test(text.replace(/\/\*([\s\S]*?)\*\//gu, ""));
```

Comment:
Can we make this a non-capturing group since it's not using the group?
```suggestion
        return /\s/u.test(text.replace(/\/\*(?:[\s\S]*?)\*\//gu, ""));
```

Reply:
Alternatively, we could use the [`s` flag](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#Advanced_searching_with_flags) (mentioning since it was mentioned in the original issue), since it's a feature that is supported in all the versions of Node we support. I don't have a strong opinion either way!

Reply:
@kaicataldo oh. yes. That would be better

Reply:
@kaicataldo I'll do that! thanks:)

Reply:
@kaicataldo  Thanks! TIL, [Advanced searching with flag](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#Advanced_searching_with_flags) is new for me 👍